scripts/carrot/BenchmarkComparisonOrchestrated.wdl

```diff
@@ -0,0 +1,346 @@
+version 1.0
+
+import "https://raw.githubusercontent.com/broadinstitute/gatk/je_carrotHCTestsScripts/scripts/carrot/BenchmarkComparison.withROCcurves.wdl" as BenchmarkAndRocCurves
```

Comment:
We can probably switch these imports to be relative now that carrot supports dependency zips.